recipes/responses/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "responses" %}
+{% set version = "0.9.0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c6082710f4abfb60793899ca5f21e7ceb25aabf321560cc0726f8b59006811c9
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv "
+
+requirements:
+  host:
```

Comment:
Is anything outside of `pip` and `python` required in the `host` section?

Reply:
probably not